actionsystemtest/lib/action_system_test/test_helper.rb

```diff
@@ -0,0 +1,17 @@
+require "capybara/dsl"
+require "action_system_test/test_helpers"
+
+module ActionSystemTest
+  module TestHelper # :nodoc:
+    include TestHelpers::ActiveJobSetup
+    include TestHelpers::Assertions
+    include TestHelpers::ScreenshotHelper
+    include Capybara::DSL
+
+    Capybara.app = Rack::Builder.new do
```

Comment:
Agree, but also, I think we should encapsulate this in a bootstrap method somewhere. It feels oddly free-floating in a helper to do this kind of global configuration. (I'm thinking something like `register_browser_driver`).